request-handler/pom.xml

```diff
@@ -63,6 +78,27 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <!--  Note: Normally, this plugin would be run using
+                "mvn compile assembly:single" but the execution
+                clause below binds it to the normal 'package' lifecycle phase
+                -->
+                <artifactId>maven-assembly-plugin</artifactId>
```

Comment:
I wasn't sure either - so it is building both at the moment.  We can decide which looks like a better way to do it.